coderd/database/queries/notifications.sql

```diff
@@ -4,7 +4,8 @@ SELECT nt.name                                                    AS notificatio
        nt.actions                                                 AS actions,
        u.id                                                       AS user_id,
        u.email                                                    AS user_email,
-       COALESCE(NULLIF(u.name, ''), NULLIF(u.username, ''))::text AS user_name
+       COALESCE(u.name, '')                                       AS user_name,
+       COALESCE(u.username, '')                                   AS user_username
```

Comment:
We should fall back to `u.username` if `u.name` is not set.

```suggestion
       COALESCE(NULLIF(u.name, ''), NULLIF(u.username, ''))::text    AS user_name,
       u.username                                                                                        AS user_username
```

Reply:
Is it a valid case that `Name` is not set?

Reply:
Yup, it's a new field I believe.

Reply:
Let me quickly patch it.

Reply:
fixed